web/src/pages/Login/Login.tsx

```diff
@@ -158,7 +147,39 @@ export default function Login() {
                                 {errorPwd} {errorEmail}
                             </p>
                             <div className="login-submit">
-                                {/* <p>Forgot password ? </p> */}
+                                <div className="forgot-password log-box" onClick={toggleResetModal}>
+                                    <span className="txt-color">
+                                        reset my password
+                                    </span>
+                            </div>
+                            <Modal
+                                isOpen={isResetModalOpen}
+                                onRequestClose={toggleResetModal}
+                                contentLabel="Reset Password Modal"
+                                style={{
+                                    overlay: {
+                                        backgroundColor: 'rgba(0, 0, 0, 0.75)', // This will create a semi-transparent black background
+                                        zIndex: 1000, // This will bring the modal to the front
+                                    },
+                                    content: {
+                                        // Add your content styles here
+                                    },
+                                }}
+                            >
+                                <h2>Reset Password</h2>
+                                <p>Please enter your email address to reset your password.</p>
+                                <input
+                                    type="email"
+                                    value={resetEmail}
+                                    onChange={(e) => setResetEmail(e.target.value)}
+                                    placeholder="Enter your email"
+                                    // Add any additional input styles
+                                />
+                                <button onClick={handlePasswordReset}>Send Email</button>
+                                <p>{resetFeedback}</p> {/* Feedback message */}
```

Comment:
you cannot just put `/* Feedback message */` here, you need to get the error message from the back-end and display it